### PR TITLE
added _I type {string} chars to {string}_ to send find-grained events

### PR DESCRIPTION
**Pull request by AlexGalichenko**
- [x] Updated the CHANGELOG.md.
- [x] Checked that there aren't other open pull requests for the same issue/update.
- [x] Checked that your contribution follows the project's contribution guidelines.
- [x] Added corresponding unit/E2E tests
- [x] Unit and E2E pass

## Description

added _I type {string} chars to {string}_ to send find-grained events

### CODE REVIEW

File: test-e2e/features/memory.feature [hunk 1]
```diff
@@ -26,7 +26,7 @@ Feature: memory
   Scenario: page title
     Then I save page title as 'memory'
     Then I expect '$memory' memory value to be equal '@qavajs'
-
+  @debug
```
Comment (KirillBogdanets) [inline test-e2e/features/memory.feature:29]: I guess it's redundant here

Comment (AlexGalichenko) [reply test-e2e/features/memory.feature]: updated :)

**Review by KirillBogdanets — APPROVED** [review]

**Review by ALegchilov — APPROVED** [review]

File: test-e2e/features/memory.feature [hunk 2]
```diff
@@ -26,7 +26,7 @@ Feature: memory
   Scenario: page title
     Then I save page title as 'memory'
     Then I expect '$memory' memory value to be equal '@qavajs'
-
+
```
Comment (ALegchilov) [inline test-e2e/features/memory.feature:29]: 👀

Comment (AlexGalichenko) [reply test-e2e/features/memory.feature:29]: :/

**Pull request merged**